packages/expo/ios/AppDelegates/ExpoAppDelegate.swift

```diff
@@ -11,7 +11,7 @@ import ReactAppDependencyProvider
  Keep functions and markers in sync with https://developer.apple.com/documentation/uikit/uiapplicationdelegate
  */
 @objc(EXExpoAppDelegate)
-open class ExpoAppDelegate: NSObject, ReactNativeFactoryProvider, UIApplicationDelegate {
+open class ExpoAppDelegate: NSObject, @preconcurrency ReactNativeFactoryProvider, UIApplicationDelegate {
```

Comment:
```suggestion
open class ExpoAppDelegate: NSObject, @MainActor ReactNativeFactoryProvider, UIApplicationDelegate {
```

not sure if that can be a mainactor? fine if not

Reply:
I don't think the syntax supports that unfortunelly 
<img width="525" height="80" alt="image" src="https://github.com/user-attachments/assets/95f937ba-476a-4a2e-b00d-f5364112bf54" />

Reply:
What we could do is to add @MainActor to the ReactNativeFactoryProvider protocol, but this triggers other errors in ExpoReactDelegate when calling `recreateRootView`
<img width="812" height="154" alt="image" src="https://github.com/user-attachments/assets/39af9c84-325f-46b9-bc1f-ac0ff9479bb8" />

Reply:
i see. thanks for the looking this over. let's keep it as-is.